### PR TITLE
Setup pnmp for form-renderer library

### PR DESCRIPTION
**Pull request by alimpens**
# Meldingen

Before opening a pull request, please ensure:

- Double-check your branch is based on `main` and targets `main`
- Code is well-commented, linted and follows project conventions

Be kind to code reviewers, please try to keep pull requests as small and focused as possible :)

### CODE REVIEW

File: .stylelintrc.json [hunk 1]
```diff
@@ -1,36 +1,30 @@
...
+      {
+        "unspecified": "bottomAlphabetical"
       }
-    }
-  ],
-  "rules": {}
+    ],
+    "property-disallowed-list": [
+      [
+        "margin",
```
Comment (Doilee) [inline .stylelintrc.json:15]: Wat is eigenlijk de reden dat deze properties disallowed zijn?
Hoe kan ik zelf handmatig padding/margin setten van bepaalde elementen?

Comment (Doilee) [reply .stylelintrc.json:15]: Ik snap dat dit niet related is aan de PR maar ik was toch  even benieuwd!

**Review by Doilee — APPROVED** [review]
Wat mij betreft ziet het er goed uit 👍

File: .stylelintrc.json (hunk 1 above)
Comment (alimpens) [reply .stylelintrc.json:15]: `margin` and `padding` are a shorthand for related directional properties, in other words `*-top`, `*-right`, `*-bottom`, `*-left`. 

In our codebase we use [logical properties](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_logical_properties_and_values), such as `padding-block` and `margin-inline-end`. This allows us to support languages that are not left-to-right.

**Pull request merged**